<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VarFuncDefDialogClass</class>
 <widget class="QDialog" name="VarFuncDefDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FalconCalc - Define Variables or Functions</string>
  </property>
  <property name="windowIcon">
   <iconset resource="FalconCalcQt.qrc">
    <normaloff>:/FalconCalcQt/Resources/calculator.ico</normaloff>:/FalconCalcQt/Resources/calculator.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>&amp;Name:</string>
        </property>
        <property name="buddy">
         <cstring>edtName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edtName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblValue">
        <property name="text">
         <string>&amp;Value:</string>
        </property>
        <property name="buddy">
         <cstring>edtValue</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edtValue"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblUnit">
        <property name="text">
         <string>&amp;Unit:</string>
        </property>
        <property name="buddy">
         <cstring>edtUnit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="edtUnit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblComment">
        <property name="text">
         <string>&amp;Comment</string>
        </property>
        <property name="buddy">
         <cstring>edtComment</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="edtComment"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblFDefinition">
        <property name="text">
         <string>function(a,b,c)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="FalconCalcQt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VarFuncDefDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VarFuncDefDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
